rules/integrations/azure/initial_access_entra_graph_single_session_from_multiple_addresses.toml

```
@@ -120,35 +144,47 @@ from logs-azure.signinlogs-*, logs-azure.graphactivitylogs-* metadata _id, _vers
     Esql.event_type_case,
     Esql.time_window_date_trunc,
```

```suggestion
    Esql.event_type_case,
```
Since this field is no longer defined it threw an error for me in the stack.